cccl

```diff
@@ -162,6 +162,14 @@ EOF
         esac
         ;;
 
+    -Wl,*)
+        IFS=',' read -ra linkopt2 <<< "$(echo $1 | sed 's/-Wl,//')"
```

Comment:
Shouldn't IFS be reset back to it's previous empty value afterwards?

Reply:
I think this sets the value for the `read` command only

Reply:
@wsfulton The environment variable is changed for this specific command only, as it's included on the same line as the command.